protostar/commands/deploy/deploy_command.py

```diff
@@ -31,23 +34,30 @@ def name(self) -> str:
 
     @property
     def description(self) -> str:
-        return "Deploys contracts."
+        return "\n".join(
+            [
+                "Deploys contracts.",
+            ]
+        )
 
     @property
     def example(self) -> Optional[str]:
-        return None
+        return "protostar deploy main -n testnet"
 
     @property
     def arguments(self) -> List[Command.Argument]:
         return [
             Command.Argument(
                 name="contract",
-                description='A name of the contract defined in protostar.toml::["protostar.contracts"]',
+                description='A name of the contract defined in protostar.toml::["protostar.contracts"].',
                 type="str",
+                is_required=True,
+                is_positional=True,
             ),
             Command.Argument(
                 name="inputs",
-                description="The inputs to the constructor",
+                short_name="i",
+                description="The inputs to the constructor.",
                 type="str",
```

Comment:
shouldn't this one be a list?

Reply:
We need an e2e test for this one

Reply:
It is. `is_array=True`.

Reply:
Added integration test instead.

Reply:
E2E is pain in the ass when used with deploying. The integration test should be enough.

Reply:
Why it's problematic

Reply:
I had to run devnet before running tests.

Now I'm confused, because the devnet fixture should work. It spawns a new process. I'll leave it to the end. The integration test should be good enough for now because it covers the fragile part, that is mapping args to the deploy method.

Reply:
It's weird. If you can't manage to make it work let me know, I will take a look on that. We need an e2e test

Reply:
I added e2e test in "Deploy command — 05" PR but it deploys to testnet instead. Deploying to devnet works, if devnet is started from the CLI level. It looks like the protostar subprocess doesn't see the devnet subprocess.

IMO, we can have one e2e test that deploys to testnet.